mithril-signer/src/message_adapters/from_epoch_settings.rs

```diff
@@ -12,9 +12,11 @@ pub struct FromEpochSettingsAdapter;
 impl TryFromMessageAdapter<EpochSettingsMessage, SignerEpochSettings> for FromEpochSettingsAdapter {
     /// Method to convert.
     fn try_adapt(message: EpochSettingsMessage) -> StdResult<SignerEpochSettings> {
+        #[allow(deprecated)]
         let epoch_settings = SignerEpochSettings {
             epoch: message.epoch,
-            signer_registration_protocol_parameters: message.next_protocol_parameters,
+            // TODO: Temporary, the value assigned will be message.signer_registration_protocol_parameters
+            signer_registration_protocol_parameters: message.next_protocol_parameters.unwrap(),
```

Comment:
This unwrap should be replaced by raising an error.

Reply:
The `unwrap` has been removed in the following [commit](https://github.com/input-output-hk/mithril/pull/2058/commits/7deee47784a311179d5231944cbeec4e691fe82c#diff-0133d53c97f55f833442e23e360d8103ed5bd26d53b110ddda903082ccd3424dR19)